canonical-web-and-design/cn.ubuntu.com
# Add 20.04 takeover and engage page + image template

## Done

Add 20.04 takeover and engage page + image template

## QA

1. download this branch
2. look at http://0.0.0.0:8010/ and http://0.0.0.0:8010/engage/ff-release-party
3. see that they look ok and match the [brief](https://docs.google.com/spreadsheets/d/1oZ95LrwJaQehseVvVrtiuLk_PvmEVuKIc5GAXmJyOpA/edit#gid=1271849439)

## Issue / Card

Fixes #7187

## Screenshots

![image](https://user-images.githubusercontent.com/441217/79992230-1f711480-84ab-11ea-84b4-f335594197c1.png)
